### ISSUE TITLE
Animate circle from one position to next

### ISSUE THREAD

**Issue by ferdicus** [nitaliano/react-native-mapbox-gl#1121]
Hi!
I would like to smoothly animate a circle/ point from one position to the other.

```
...
<Mapbox.Animated.ShapeSource
  id='shapeLayer'
  shape={Mapbox.geoUtils.makePoint(this.state.position)}
>
  <Mapbox.Animated.CircleLayer
    id='circleLayer'
    style={layerStyles.circle}
  />
</Mapbox.Animated.ShapeSource>
...
```

My assumption is, that if I use the `Animated` version of `CircleLayer` and `ShapeSource`, that it will play nice with an animated value.

However I'm unsure how to transform this: `Mapbox.geoUtils.makePoint(this.state.position)` into an animated value that interpolates :S

I'm grateful for any pointers - thanks in advance

**Comment by ferdicus** [comment]
*pump* :)

**Comment by nitaliano** [comment]
We have an example of this https://github.com/mapbox/react-native-mapbox-gl/blob/master/example/src/components/DriveTheLine.js

**Comment by ferdicus** [comment]
hey, @nitaliano thanks for your reply.

I find the example pretty hard to parse and it seems like overkill for my usecase. 
I simply want to move a circle from one spot to the next in a straight line - no need to move along a route. 

Can you maybe give me some pointers?

Again, thanks in advance.

**Comment by mrgreenish** [comment]
@ferdicus Did you manage to do this in a other way?

**Comment by GislainMoreira** [comment]
I’m also interested !

**Comment by hofa77** [comment]
I could do it in a dirty way !
put your code where the coordinate of state is being change in a block of  

> requestAnimationFrame

and then change yout state 
then start a simple animation !
`
 
requestAnimationFrame(() => {

// here I am changing the coordinate in the state
        this.state.featureCollectionEnd.features[0].geometry.coordinates = center;
        this.setState({
          featureCollectionEnd: this.state.featureCollectionEnd
        });

// here I am running a simple animation 
        this._animatedValue = new Animated.Value(0);
        this._anim = Animated.timing(this._animatedValue, {
          toValue: 1,
          duration: 0,
          useNativeDriver: false
        });

        this._anim.start();
      });
`

**Comment by VladMatvei** [comment]
Any update on this? The DriveTheLine example is an overkill

**Comment by ferdicus** [comment]
> Any update on this? The DriveTheLine example is an overkill

Please check the README of this repo - it has been deprecated. 
Check out https://github.com/react-native-mapbox-gl/maps instead
